### PR TITLE
Extract original driver from decorated driver when Augmenting a driver

### PR DESCRIPTION
**Pull request by baflQA**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Extract the original driver from the decorated driver.

### Motivation and Context
When trying to use the Augmentor on a decorated driver, the extractRemoteWebDriver method will be executed infinitely. 

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by diemol — COMMENTED** [review]
Thanks for this PR.

Is there a way we can reproduce the issue first to understand the fix? Maybe it can be added as a test?

**Comment by baflQA** [comment]
> Thanks for this PR.
> 
> Is there a way we can reproduce the issue first to understand the fix? Maybe it can be added as a test?

I will be happy to add a test, once I'm finished with indexing the project 😂 it was running the whole night and crashed my IDE at some point...

**Comment by titusfortner** [comment]
Make sure you mark as excluded all the `bazel-*` directories and the `build` directory. InteliJ hates indexing Bazel generated things for some reason.

**Comment by baflQA** [comment]
> Make sure you mark as excluded all the `bazel-*` directories and the `build` directory. InteliJ hates indexing Bazel generated things for some reason.

That helped a lot! Thx

**Comment by titusfortner** [comment]
You have *no idea how long it took me to figure out what was causing the problem...

File: java/src/org/openqa/selenium/support/BUILD.bazel [hunk 1]
```diff
@@ -18,6 +18,7 @@ java_export(
         ":page-factory",
         "//java/src/org/openqa/selenium/support/events",
         "//java/src/org/openqa/selenium/support/locators",
+        "//java/src/org/openqa/selenium/support/decorators",
```
Comment (diemol) [inline java/src/org/openqa/selenium/support/BUILD.bazel]: Can you please leave this in order? Should go after `events`.

File: java/src/org/openqa/selenium/support/decorators/BUILD.bazel [hunk 2]
```diff
@@ -7,6 +7,7 @@ java_library(
     visibility = [
         "//java/src/org/openqa/selenium/support:__subpackages__",
         "//java/test/org/openqa/selenium/support/decorators:__pkg__",
+        "//java/src/org/openqa/selenium/remote:__subpackages__",
```
Comment (diemol) [inline java/src/org/openqa/selenium/support/decorators/BUILD.bazel]: Can you please leave this in order? Should go before the `/test/` one.

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by diemol — APPROVED** [review]
Thank you, @baflQA!

**Pull request merged**